third_party/terraform/resources/resource_google_project_service.go

```diff
@@ -106,38 +100,29 @@ func resourceGoogleProjectServiceDelete(d *schema.ResourceData, meta interface{}
 	config := meta.(*Config)
 
 	if disable := d.Get("disable_on_destroy"); !(disable.(bool)) {
-		log.Printf("Not disabling service '%s', because disable_on_destroy is false.", d.Id())
+		log.Printf("[WARN] Project service %q disable_on_destroy is false, skip disabling service", d.Id())
 		d.SetId("")
 		return nil
 	}
 
-	id, err := parseProjectServiceId(d.Id())
```

Comment:
Why do we use the `id` in `Read` but use `getProject` here?

Reply:
we use the ID in read because we don't necessarily have project set in the resource prior to Read (i.e. we only call (d.Set("project", project) in Read() - I didn't realize it and initially removed the ID parsing code and ran into that issue, so I added the parsing ID back, but I can either:

- switch it back to parsing ID here in Delete() or 
- actually set project in create and use getProject in Read().

Reply:
The only case where we would only have the `id` set and not `project` would be during import, right? Could we upgrade the import fn to set `project` and use `getProject` for consistency?

Reply:
i'm ok with that